sdk/compute/azure-mgmt-vmware/setup.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import re
+import os.path
+from io import open
+from setuptools import find_packages, setup
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-mgmt-vmware"
+PACKAGE_PPRINT_NAME = "Vmware Management"
```

Comment:
Where does this show up? May be `AVS Management` or `Azure VMware Solution Management`. See how it is referred to in the REST API docs https://docs.microsoft.com/en-us/rest/api/avs/